packages/block-library/src/gallery/gallery-image.js

```diff
@@ -18,9 +18,9 @@ class GalleryImage extends Component {
 	constructor() {
 		super( ...arguments );
 
-		this.onImageClick = this.onImageClick.bind( this );
+		this.onSelectImage = this.onSelectImage.bind( this );
 		this.onSelectCaption = this.onSelectCaption.bind( this );
-		this.onKeyDown = this.onKeyDown.bind( this );
+		this.onRemoveImage = this.onRemoveImage.bind( this );
 		this.bindContainer = this.bindContainer.bind( this );
```

Comment:
`bindContainer` is unused and should be removed.

Reply:
mmm, notice that it's used by the `<img>` to store a reference. Then that reference is used in `onRemoveImage`.